_install-and-configure/configuration/search-settings.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: default
+title: Search settings
+parent: Configuring OpenSearch
+nav_order: 80
+---
+
+# Search settings
+
+OpenSearch supports the following search settings:
+
+- `search.max_buckets` (Dynamic, integer): The maximum number of aggregation buckets allowed in a single response. Default is `65535`. 
+
+- `search.phase_took_enabled` (Dynamic, Boolean): Enables returning phase-level `took` time values in search responses. Default is `false`. 
+
+- `search.allow_expensive_queries` (Dynamic, Boolean): Allows or disallows expensive queries. For more information, see [Expensive queries]({{site.url}}{{site.baseurl}}/query-dsl/index/#expensive-queries).
+
+- `search.default_allow_partial_results` (Dynamic, Boolean):  A cluster-level setting that allows returning partial search results if a request times out or a shard fails. If a search request contains an `allow_partial_search_results` parameter, the parameter takes precedence over this setting. Default is `true`. 
+
+- `search.cancel_after_time_interval` (Dynamic, time unit): A cluster-level setting that specifies the maximum amount of time that a search request can run before it is canceled at the shard level. After this time has been reached, a request is stopped and all associated tasks are canceled. Default is `-1`.
+
+- `search.default_search_timeout` (Dynamic, time unit): A cluster-level setting that sets the default timeout for all search requests at the coordinating node level. If the `timeout` is specified in the search request, it takes precedence over this setting. Default is `-1` (no timeout).
+
+- `search.default_keep_alive` (Dynamic, time unit): Specifies the default keep alive value for scroll and Point in Time (PIT) searches. Because a request may land on a shard multiple times (for example, during the query and fetch phases), OpenSearch opens a _request context_ that exists for the full duration of the request, to ensure consistency of the shard state for each individual shard request. In a standard search, once the fetch phase completes, the request context is closed. For a scroll or a PIT search, OpenSearch keeps the request context open until explicitly closed (or until the keep alive time is reached). A background thread periodically checks all open scroll and PIT contexts and deletes the ones that have exceeded their keep alive timeout. The `search.default_keep_alive` setting specifies how frequently the contexts are checked for expiration. Default is `5m`.
```

Comment:
May be worth mentioning that a scroll or PIT request can explicitly specify the keep alive, which will take precedence over this setting.

> The `search.default_keep_alive` setting specifies how frequently the contexts are checked for expiration.

The `search.keep_alive_interval` setting specifies how frequently they're checked for expiration. The `search.default_keep_alive` setting is the default deadline for expiration.

Reply:
Thank you! Added.